### PR TITLE
Adds a plushy crate

### PR DESCRIPTION
**Pull request by TeomanTheGreat**
## About The Pull Request
Adds a crate which contains 5 random plushies 

## Why It's Good For The Game
Plushies are neat and cargo should be able to buy them

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image
image
(broken tile from ratvar and nar'sie spawning in the same box)
</details>

## Changelog
:cl: Phil Smith
add: Cargo can now order 5 plushies in a crate
/:cl:

### CODE REVIEW

**Comment by Dejaku51** [comment]
Cargo can buy plushes in Toy Crate

**Comment by TeomanTheGreat** [comment]
> Cargo can buy plushes in Toy Crate

the toy crate has RNG (like 50 percent chance per plush). this pr is just flat out 5.

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -2782,6 +2782,22 @@
 	)
 	crate_type = /obj/structure/closet/crate/wooden
 
+
+/datum/supply_pack/costumes_toys/randomised/plush
+	name = "Plushy Crate"
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	name = "Plushie Crate"
```
Plushy is an adjective for the fabric, plushie is a noun for the toy.

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -2782,6 +2782,22 @@
...
+	desc = "A crate filled with 5 plushies!"
+	cost = 1500
+	num_contained = 5
+	contains = list()
+	crate_type = /obj/structure/closet/crate/wooden
+	crate_name = "plushy crate"
```
Comment (ike709) [inline code/modules/cargo/packs.dm]:
```suggestion
	crate_name = "plushie crate"
```

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**